config/schema_descriptions.yml

```diff
@@ -0,0 +1,72 @@
+reporting_patient_states:
+  description: "Monthly summary of a patient's information and health indicators. This table has one row per patient, per month, from the month of the patient's registration."
+  columns:
+    patient_id: "ID of the patient"
+    recorded_at: "Time (in UTC) at which the patient was registered"
+    status: "active, dead, migrated, etc"
+    gender: "Gender of the patient"
+    age: "Age of the patient as entered by a nurse"
+    age_updated_at: "Time (in UTC) at which the age was last updated"
+    date_of_birth: "Date of birth of the patient"
+    current_age: "Computed age as of today, given age, age-updated-at and date-of-birth. This will have the same value for a given patient across all rows."
+    month_date: "Date at the beginning of the month for which this row has the latest information of a patient"
+    month: "Month (1-12) of year"
+    quarter: "Quarter (1-4) of year"
+    year: "Year in YYYY format"
+    month_string: "String that represents a month, in YYYY-MM format"
+    quarter_string: "String that represents a month, in YYYY-Q format"
+    hypertension: "Yes if patient is hypertensive, no if not, and nil if the data is unavailable"
+    prior_heart_attack: "Yes if the patient has had a heart attack, no if not, nil if unknown"
+    prior_stroke: "Yes if the patient has had a stroke, no if not, nil if unknown"
+    chronic_kidney_disease: "Yes if the patient has had a chronic kidney disease, no if not, nil if unknown"
+    receiving_treatment_for_hypertension: "Yes if the patient was already receiving treatment for hypertension during registration"
+    diabetes: "Yes if patient is diabetic, no if not, and nil if the data is unavailable"
+    assigned_facility_id: "Facility ID to which the patient is currently assigned"
+    assigned_facility_size: "Size of the assigned facility"
```

Comment:
Consider adding "patient's" everywhere. "...patient's assigned facility"

Reply:
Added this. It's a bit awkward in some places: "patient's registration facility's state", but it's alright I think.